src/core/main.js

```diff
@@ -62,6 +62,10 @@ class p5 {
       mousemove: null,
       mousedown: null,
       mouseup: null,
```

Comment:
Do these and the touch ones below still need to be here?

Reply:
Oh yes! I missed these during the cleanup. But I think we need the touch ones for the `touches[]` array since we're handling multiple touches with the touch events. I've removed the mouse events.

Reply:
I wonder if it would be possible to implement `touches[]` with pointer events as well, essentially caching the events as they come in, perhaps something like described here: https://developer.mozilla.org/en-US/docs/Web/API/Pointer_events/Multi-touch_interaction

Reply:
If the local test sketches you tried are working as intended then it should be fine for now, you can try to fix the tests if it is easy to do but that's not a priority as I may need to rewrite them later.

Reply:
I realized later that if I implement touches with pointer-events then `onpointerup()/down()/move()` then in both mouse.js and touch.js and `mousePressed()/Released()` stops working. I think this is because for the sketches these events fire from both touch.js and mouse.js parallel. So, maybe the choice lies between either having one file `pointer.js` which would have pointer events for touches and mouse, or let the touches be decoupled by using touch events.

Reply:
Yes, let's go with just one file `pointer.js`

Reply:
I've tested all the examples and they work fine.

Reply:
> Yes, let's go with just one file `pointer.js`

I've tested the examples and they seem to work fine.